packages/quantic/cypress/integration/summary/summary.cypress.ts

```diff
@@ -0,0 +1,91 @@
+import {configure, reset} from '../../page-objects/configurator';
+import {
+  InterceptAliases,
+  interceptSearch,
+  interceptSearchIndefinitely,
+} from '../../page-objects/search';
+import {scope} from '../../reporters/detailed-collector';
+import {SummaryExpectations as Expect} from './summary-expectations';
+import {ResultsPerPageActions} from '../results-per-page/results-per-page-actions';
+
+describe('quantic-summary', () => {
+  const summarytUrl = 's/quantic-summary';
+
+  function visitSummary(waitForSearch = true) {
+    interceptSearch();
+    cy.visit(summarytUrl);
+    configure();
+    if (waitForSearch) {
+      cy.wait(InterceptAliases.Search);
+    }
+  }
+
+  function setupWithPauseBeforeSearch() {
+    interceptSearchIndefinitely();
+    cy.visit(summarytUrl);
+    configure();
+  }
+
+  function loadFromUrlHash(urlHash: string) {
+    interceptSearch();
+
+    cy.visit(`${summarytUrl}#${urlHash}`);
+    configure();
+  }
+
+  it('should not render before results have returned', () => {
+    setupWithPauseBeforeSearch();
+
+    Expect.displaySummary(false);
+  });
+
+  describe('when loading default summary', () => {
+    it('should work as expected', () => {
+      visitSummary();
+
+      Expect.displaySummary(true);
+      Expect.displayRange(true);
+      Expect.displayTotal(true);
```

Comment:
It would be nice to validate that the total (at least) is properly formatted. To ensure the test is reliable, we might want to intercept the search response and force the total count value.